tests/testAutopas/tests/selectors/AutoTunerRemainderTraversalTest.cpp

```diff
@@ -12,9 +12,298 @@
 #include "autopas/selectors/tuningStrategy/FullSearch.h"
 #include "testingHelpers/commonTypedefs.h"
 
+/**
+ * Can test all individual steps of iterate pairwise. Expects exactly two particles separated by 1.
+ *
+ * @note Tests invoking this function should have AutoPas logger instantiated (e.g. by inheriting from AutoPasTestBase).
+ *
+ * @note Buffers need to have at least one (empty) cell. They must not be empty.
+ *
+ * @param particlesContainerOwned
+ * @param particlesBuffer
+ * @param particlesHaloBuffer
+ * @param n3 Newton3 on or off
+ */
+void testIteratePairwiseSteps(std::vector<Molecule> &particlesContainerOwned,
+                              std::vector<Molecule> &particlesContainerHalo,
+                              std::vector<autopas::FullParticleCell<Molecule>> &particlesBuffers,
+                              std::vector<autopas::FullParticleCell<Molecule>> &particlesHaloBuffers,
+                              autopas::Newton3Option n3) {
+  // sanity check that there are exactly two particles in the test
+  const auto numParticlesBuffers =
```

Comment:
Rename `numParticlesBuffers` to `numParticlesInBuffers` for clarity. Similarly `numParticlesInHaloBuffers`. You _could_ also rename `particlesBuffers` to `particlesInBuffers` for consistency - but I don't think this is that important.

Reply:
`particlesBuffers` are the actual buffers so I think this name makes more sense?

Reply:
Yes I see. But I still think the `numParticlesBuffers` should be `numParticlesInBuffers` to avoid confusion that this variable does not hold the number of particle buffers.

Reply:
done 6685dbbb05334e01eb1001ad4b603bcceed1b05b